src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -637,3 +637,42 @@ def test_fw_codeblock():
         "    a=1\n"
         "PRINT *, 'I am a code block'\n"
         "    PRINT *, 'with more than one line'\n" in result)
+
```

Comment:
pylint complains about a missing absolute_import and an Import alias not renaming original package. Neither of those are the fault of this PR though.

Reply:
Again I don't see these in either pylint 1.9.5 (python2) or pylint 2.3.1 (python3). I've added an absolute import and will hope the renaming problem is the same issue as the previous comment.

Reply:
Code now scores 10.0 :-)